pkgs/development/interpreters/python/wrap.sh

```diff
@@ -35,6 +53,33 @@ patchPythonScript() {
     sed -i "$f" -re '@magicalSedExpression@'
 }
 
+_wrapPythonProgram() {
+    # wrapProgram creates the executable shell script described
+    # above. The script will set PYTHONPATH and PATH variables.!
+    # (see pkgs/build-support/setup-hooks/make-wrapper.sh)
+    local -a wrap_args=("$f"
+                    --prefix PATH ':' "$program_PATH"
+                    )
+
+    if [ -z "$permitUserSite" ]; then
+        wrap_args+=(--set PYTHONNOUSERSITE "true")
+    fi
+
+    # Add any additional arguments provided by makeWrapperArgs
+    # argument to buildPythonPackage.
+    # We need to support both the case when makeWrapperArgs
+    # is an array and a IFS-separated string.
+    # TODO: remove the string branch when __structuredAttrs are used.
+    if [[ "${makeWrapperArgs+defined}" == "defined" && "$(declare -p makeWrapperArgs)" =~ ^'declare -a makeWrapperArgs=' ]]; then
```

Comment:
`makeWrapperArgs` would disappear, because we will have the `appsWrapperArgs`.

There is some code here for backwards-compatibiltiy as well that will be removed.